misc/sync-typeshed.py

```diff
@@ -185,7 +185,18 @@ def main() -> None:
         "9f3bbbeb1",  # ParamSpec for functools.wraps
     ]
     for commit in commits_to_cherry_pick:
-        subprocess.run(["git", "cherry-pick", commit], check=True)
+        try:
+            subprocess.run(["git", "cherry-pick", commit], check=True)
+        except subprocess.CalledProcessError:
+            if sys.__stdin__.isatty():
+                # Allow the option to merge manually
+                print(
+                    f"Commit {commit} failed to cherry pick."
+                    " In a separate shell, please manually merge and continue cherry pick."
+                )
+                input("Did you finish the cherry pick?")
```

Comment:
Trying the script out locally, my naive expectation was that I was required to either type "y" or "n" to continue. I typed in "y", which meant that the line on the terminal was

```
Did you finish the cherry pick?y
```

Looking at the script again, you obviously _don't_ need to type anything at all. But it still might be nice to have a space at the end here so that it doesn't look so weird if a user _does_ input something 😆

```suggestion
                input("Did you finish the cherry pick? ")
```

Reply:
Yeah, It was a quick change on my part to get feedback if it's useful. I'll actually check the return of `input` 😄

Reply:
Done!